rotkehlchen/chain/ethereum/modules/compound/utils.py

```diff
@@ -0,0 +1,17 @@
+from typing import Optional
+from rotkehlchen.assets.asset import CryptoAsset, EvmToken
+from rotkehlchen.chain.ethereum.modules.compound.constants import CTOKEN_MAPPING
+
+
+def get_compound_underlying_token(token: EvmToken) -> Optional[CryptoAsset]:
+    """
+    Return the underlying token for a compound token. If the provide token is
+    unknown it returns None.
+
+    May raise:
+    - UnknownAsset: If the underlying asset can't be resolved
```

Comment:
Hm .... I don't think this is a good interface.

You either:
- Always return and in case of error, raise
- Do not raise, but in case of any error return None.

Choose one. Why? In the 3 places you use it you also end up having 3 extra if checks. So you check both for exception and if it's None. Makes sense?